scripts/upgrade-hooks/1.1.0.ts

```diff
@@ -0,0 +1,251 @@
+import { createClient, fetchExchange } from "@urql/core";
+import { AbiCoder, encodeBytes32String } from "ethers";
+import hre from "hardhat";
+import fetch from "node-fetch";
+import fs from "fs";
+import path from "path";
+
+const { ethers } = hre;
+const VERSION = "1.1.0";
+
+const abiCoder = new AbiCoder();
+
+// Function to get the correct GRAPHQL_URL based on chainId and env
+function getGraphQLUrl(chainId: number, env: string): string {
+  const subgraphConfig = JSON.parse(fs.readFileSync(path.join(__dirname, "subgraph_config.json"), "utf8"));
+
+  if (!subgraphConfig[env] || !subgraphConfig[env][chainId.toString()]) {
+    throw new Error(`No subgraph configuration found for chainId ${chainId} and env ${env}`);
+  }
+
+  const url = subgraphConfig[env][chainId.toString()].subgraphUrl;
+  if (url === "TBD") {
+    throw new Error(`Subgraph URL is TBD for chainId ${chainId} and env ${env}`);
+  }
+
+  return url;
+}
+
+// Function to get the InitializationFacet address based on chainId and env
+function getInitializationFacetAddress(chainId: number, env: string): string {
+  const addressesDir = path.join(__dirname, "..", "..", "addresses");
+  const addressFile = fs.readFileSync(path.join(addressesDir, `${chainId}-${env}.json`), "utf8");
+  const addressData = JSON.parse(addressFile);
```

Comment:
Not a correct path in `addressFile`, since the addresses filename also includes `network`.

Also, I suggest using 
```const addressData = await readContracts(env)```
Where `readContracts` is from `libraries/utils` which does the job and also handles some special cases for filepaths.